lib/internal/net.js

```diff
@@ -10,5 +10,6 @@ function isLegalPort(port) {
 }
 
 module.exports = {
-  isLegalPort
+  isLegalPort,
+  kNormalizedConnectSymbol: Symbol('normalizedConnect')
```

Comment:
It's also used by `listen()`. Perhaps we should just call it `normalizedArgsSymbol` with a `'normalizedArgs'` symbol value? Keeping it generic may also help if we ever need/want it for other normalization functions, like the ones in `child_process` or something.